src/mame/includes/klax.h

```diff
@@ -58,4 +57,38 @@ class klax_state : public driver_device
 	static const atari_motion_objects_config s_mob_config;
 };
 
+class klax_bootleg_state : public klax_state
+{
+public:
+	klax_bootleg_state(const machine_config &mconfig, device_type type, const char *tag)
+		: klax_state(mconfig, type, tag)
+		, m_audiocpu(*this, "audiocpu")
+		, m_msm(*this, "msm%u", 1)
+		, m_rombank(*this, "rombank")
+		, m_audio_ram(*this, "audioram")
+	{ }
+
+	void klax5bl(machine_config &config);
+
+private:
+	virtual void machine_start() override;
```

Comment:
This should be `protected` rather than `private` to make life easier if another state class ends up being derived from this.  (You want to be able to call the base `machine_start` to register for save states, etc.).